src/projectM-sdl/pmSDL.cpp

```diff
@@ -144,7 +167,10 @@ void projectMSDL::keyHandler(SDL_Event *sdl_evt) {
         case SDLK_f:
             if (sdl_mod & KMOD_LGUI || sdl_mod & KMOD_RGUI || sdl_mod & KMOD_LCTRL) {
                 // command-f: fullscreen
-                toggleFullScreen();
+				// Stereo requires fullscreen
+#ifndef STEREOSCOPIC_SBS
```

Comment:
maybe this check would be better in `toggleFullScreen()` - just make it always toggle to true or no-op if stereo

Reply:
Ok i will rework these